go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	golang.org/x/net v0.0.0-20210825183410-e898025ed96a
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
 	gopkg.in/fsnotify.v1 v1.4.7
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
We can avoid the additional direct dependency by sticking to `github.com/ghodss/yaml` for the new code.